packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -180,7 +180,7 @@ class SystemRulesTable extends React.Component {
         cells: [
             title,
             profile.name,
-            severity,
+            severity.charAt(0).toUpperCase() + severity.slice(1),
```

Comment:
Why not just make it all lowercase? Or, even if you want it capitalized, why not apply a CSS class with `text-transform: capitalize;`? If it's just for the purposes of presentation, I think the CSS style makes more sense.